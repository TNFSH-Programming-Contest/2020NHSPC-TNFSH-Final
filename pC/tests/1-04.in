5 6 2
3 4 120110
3 5 294846
1 4 926683
2 4 397938
1 3 807671
5 4 861280
