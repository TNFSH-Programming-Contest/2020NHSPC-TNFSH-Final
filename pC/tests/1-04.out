2649242 0 1036096 397938 2438682 
